cspell/.cspell.json

```diff
@@ -12,11 +13,13 @@
     "**/*.html",
     "**/*.m4a",
     "**/*.mp3",
+    "**/*.mp4",
     "**/*.onnx",
     "**/*.osm",
     "**/*.pcd",
     "**/*.pt",
     "**/*.stl",
+    "**/*.svg",
```

Comment:
Please refer to this comment. https://github.com/tier4/autoware-spell-check-dict/pull/295#discussion_r900063836

```suggestion
```